bin/config.js

```diff
@@ -64,6 +64,9 @@ module.exports = {
   caching: {
     service: loadFactory(config.get('CACHING_PROVIDER') || 'memory', 'caching')
   },
+  cdn: {
+    service: loadFactory('cf', 'cdn')
```

Comment:
want to spell out `cloudflare` here instead of using `cf`?

Reply:
I just didn’t want anyone to find out product searching for the for-profit company, when it doesn’t participate in it afaik.